skins/laika/src/components/Question.vue

```diff
@@ -51,3 +51,25 @@ export default Vue.extend( {
 	},
 } );
 </script>
+
+<style lang="scss">
+@import "../scss/custom";
```

Comment:
why is this part of this PR?

Reply:
The CSS was in the wrong place before and I moved it when adding my own. Definitely unrelated, yes :bowing_man:.